### ISSUE TITLE
Is there a Napkin-like book for physics?

### ISSUE THREAD

**Issue by fzyzcjy** [vEnhance/napkin#194]
Hi thank you for the book, which is quite helpful and I enjoy reading it! I wonder whether there exists a Napkin-like book for physics in the world?

**Comment by vEnhance** [comment]
I'm not aware of anything out there right now. Perhaps others can chime in if they know of anything.

**Comment by fzyzcjy** [comment]
Thank you all the same! Hope someone knows such a book 👀

**Comment by will-lancer** [comment]
No, there isn't. Not like Evan has written the Napkin. There are survey books of physics, but most of them are old. One that springs to mind is Georg Joos' "[Theoretical Physics](https://archive.org/details/in.ernet.dli.2015.285908/page/n21/mode/2up)" or the [Feynman Lectures on Physics](https://www.feynmanlectures.caltech.edu) (covers first two years of undergrad). If you want something for the physics olympiad that'll also teach you the first 2-3 years of undergrad physics, then check out Kevin Zhou's [handouts](https://knzhou.github.io/#teaching). No one has really written a big survey book on advanced physics afaik. I might write one over the coming years lol so maybe check my account in 2025 to see if I cooked something up.

**Comment by fzyzcjy** [comment]
Thank you for the information!

Btw I am learning https://theoreticalminimum.com/ currently (It is thicker than Napkin, but teaches quite well)

**Comment by zaaahir** [comment]
Is there a Napkin like book for any other sciences (biology, chemistry), or for other applied areas (such as statistics)?

**Comment by alexhsu-nlp** [comment]
> Is there a Napkin like book for any other sciences (biology, chemistry), or for other applied areas (such as statistics)?

Hi zaaahir,

For chemistry, it probably depends on how you would like to approach the field. If you want a particular "napkin" that is concise (I mean <2000 pages) and has a clear logical narrative that is easy to remember, I'm afraid there's none (at least within my reach). Many subfields of chemistry are quite descriptive and inevitably involve massive fact-listing, and many of these facts cannot be effectively summarized into general rules; when you compress them into a concise introduction, you just lose these pieces of information. I would just suggest fetching one textbook for each of the following as a start (and skip any unimportant details you feel boring, especially for inorganic and biochem): inorganic chemistry, organic chemistry, physical chemistry, quantum chemistry, biochemistry. Other things like analytical chemistry basically only count when you are *in* the field.

Meanwhile, as an unrelated suggestion, whenever I read an introduction to a general field, I will forget almost everything after a year or two. I start to remember something in the general field only when I start to read the intro to its subfields.

**Comment by zaaahir** [comment]
Slight bump again to ask if anyone has suggestions for theoretical CS?

**Comment by lianghu** [comment]
For Physics, this it is 
An Introduction to the Meaning and Structure of Physics  [Leon N Cooper]
1968
ISBN: 9780060413552

**Comment by amitlevy** [comment]
I personally think the Feynman Lectures on Physics are exactly that

**Comment by will-lancer** [comment]
There actually is! The user "Chiral Anomaly" on PhysicsSE has a website where they post tons of clear, technical articles about neat physical ideas. It's not "the physics Napkin", per se, but it's the closest thing I've ever seen. [Here](https://www.cphysics.org) is the website; enjoy!

**Comment by fzyzcjy** [comment]
Thanks friends for the suggestions!

**Comment by SingularHomology** [comment]
Would Jakob Schwichtenberg's [No-Nonsense Books](https://nononsensebooks.com/#books) qualify? They cover classical mechanics, QM, EM and even QFT.

**Comment by will-lancer** [comment]
Schwichtenberg's No-Nonsense books definitely do not qualify. They are not very good books and don't cover enough content. The closest thing you can get to a "one-stop-shop" for physics are David Tong's notes, and for more advanced topics, John McGreevy's website. Knzhou's notes also have lots of references in their first few pages that would be useful to look at.
